edit-post/components/plugins-panel/index.js

```diff
@@ -44,7 +45,7 @@ function PluginsPanel( { onClose, plugin } ) {
 				/>
 			</div>
 			<div className="edit-post-plugins-panel__content">
```

Comment:
It seem like `PluginsPanel` should be converted into the slot.

Reply:
Yes, that's the plan.